internal/godocfx/main.go

```diff
@@ -58,42 +58,43 @@ func main() {
 	outDir := flag.String("out", "obj/api", "Output directory (default obj/api)")
 	projectID := flag.String("project", "", "Project ID to use. Required when using -new-modules.")
 	newMods := flag.Bool("new-modules", false, "Process all new modules with the given prefix. Uses timestamp in Datastore. Stores results in $out/$mod.")
+	// TODO: flag to set output URL path
 
 	log.SetPrefix("[godocfx] ")
 
 	flag.Parse()
-	if flag.NArg() != 1 {
+	if flag.NArg() == 0 {
 		log.Fatalf("%s missing required argument: module path/prefix", os.Args[0])
 	}
 
-	mod := flag.Arg(0)
+	modNames := flag.Args()
 	var mods []indexEntry
 	if *newMods {
 		if *projectID == "" {
 			log.Fatal("Must set -project when using -new-modules")
 		}
 		var err error
-		mods, err = newModules(context.Background(), indexClient{}, &dsTimeSaver{projectID: *projectID}, mod)
+		mods, err = newModules(context.Background(), indexClient{}, &dsTimeSaver{projectID: *projectID}, modNames)
 		if err != nil {
 			log.Fatal(err)
 		}
 	} else {
-		modPath := mod
-		version := "latest"
-		if strings.Contains(mod, "@") {
-			parts := strings.Split(mod, "@")
-			if len(parts) != 2 {
-				log.Fatal("module arg expected only one '@'")
+		for _, mod := range modNames {
+			modPath := mod
+			version := "latest"
+			if strings.Contains(mod, "@") {
+				parts := strings.Split(mod, "@")
+				if len(parts) != 2 {
+					log.Fatal("module arg expected only one '@'")
+				}
+				modPath = parts[0]
+				version = parts[1]
 			}
-			modPath = parts[0]
-			version = parts[1]
-		}
-		modPath = strings.TrimSuffix(modPath, "/...") // No /... needed.
-		mods = []indexEntry{
-			{
+			modPath = strings.TrimSuffix(modPath, "/...") // No /... needed.
```

Comment:
Unclear to me end to end how module strings are moving through the system, do you ever need to sanitize trailing comments?

Reply:
I'm not sure what you mean by trailing comments. This comes from a command line argument. This makes sure that if the user _did_ include the `/...` at the end, we handle it transparently. A previous version of this command passed the argument through directly, and I made use of `/...`, so I didn't want to break that.